src/stores/RedisStore.ts

```diff
@@ -0,0 +1,40 @@
+import { connect, RedisConnectOptions } from "https://deno.land/x/redis@v0.25.0/mod.ts"
+import type { Ratelimit } from "../types/types.d.ts";
+import { Store } from "./AbstractStore.ts";
+
+export class RedisStore extends Store {
+    private readonly store;
+
+    constructor(options: RedisConnectOptions) {
+        super();
+        this.store = connect({
+            hostname: options.hostname,
+            port: options.port,
+            tls: options.tls,
+            db: options.db,
+            password: options.password,
+            name: options.name,
+            maxRetryCount: options.maxRetryCount,
+            retryInterval: options.retryInterval,
+        })
+    }
+
+    public async get(ip: string) {
+        const data = await (await this.store).get(ip)
+        if (!data) return
+        return JSON.parse(data)
+    }
+
+    public async set(ip: string, ratelimit: Ratelimit) {
+        (await this.store).set(ip, JSON.stringify(ratelimit));
```

Comment:
The `await` doesn't need to be inside `()` remove `()`.

Reply:
This must also return the new value as callback, you could pretty much just return this statement.

Reply:
![image](https://user-images.githubusercontent.com/59208097/147293186-03c774d3-d274-4c0b-a65d-7a20fa96bd1c.png)
when i remove the brackets this error comes up

> The `await` doesn't need to be inside `()` remove `()`.